crates/starknet-server/src/main.rs

```diff
@@ -57,7 +64,22 @@ async fn main() -> Result<(), anyhow::Error> {
         IpAddr::from_str(starknet_config.host.as_str()).expect("Invalid value for host IP address");
     let mut addr = SocketAddr::new(host, starknet_config.port);
 
-    let api = api::Api::new(Starknet::new(&starknet_config)?);
+    // Load starknet transactions
+    let mut transactions: StarknetTransactions = StarknetTransactions::default();
```

Comment:
Let's move this to separate function

Reply:
Fixed in 3028e91e1f57b71f5d9daae23a39ce7bfbce8b90